db/migrations/002_add_link_model_to_attribute.rb

```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
```

Comment:
Now that https://github.com/mountetna/magma/pull/140 has been merged, there's already a migration version 002 ([`db/migrations/002_add_validation_to_attributes.rb`](https://github.com/mountetna/magma/blob/4880aaf8c2720465a84f8fed52a8dc5179f17316/db/migrations/002_add_validation_to_attributes.rb)). Sequel won't let us have two migrations with the same version, so this file will have to be renamed to 003.